css/about.css

```diff
@@ -128,11 +126,12 @@ Media Queries (overriding styles for different screen sizes)
 		left: 0;
 		width: 100%;
 		height: 100%;
-		background: linear-gradient(107deg,
-				rgba(9, 9, 9, 0.8) 9.51%,
-				rgba(9, 9, 9, 0) 76.72%);
+		background: linear-gradient(
+			107deg,
+			rgba(9, 9, 9, 0.8) 9.51%,
+			rgba(9, 9, 9, 0) 76.72%
+		);
 		z-index: 1;
-
 	}
 
 	.hero-text {
```

Comment:
The text alignment in the hero  doesn't seem to be working on desktop

<img width="551" alt="Screen Shot 2024-01-30 at 11 37 52 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/225275e4-89f0-4223-870a-58cc8c3ea65d">

Reply:
text has been fixed